### DISCUSSION TITLE
Invalid device link

### DISCUSSION THREAD

**Discussion by romanmendelproject** [netbox-community/netbox#7912] [Q&A] [answered]
Good day!
Netbox version 3.0.10
In the rack, when clicking on a device, it links to the netbox: 8080 instead of a direct link to the device, is that how it should be?
![image](https://user-images.githubusercontent.com/33754158/142997610-ab338cb0-076a-4e88-b9be-c3eb5e38e87a.png)
![image](https://user-images.githubusercontent.com/33754158/142998119-fecf1c5c-394e-408b-84d9-4b4e2df626fd.png)

**Comment by candlerb** [comment, marked as answer]
This means you've not configured your proxy (apache or nginx) correctly.  Copy the sample configs [here](https://github.com/netbox-community/netbox/tree/v3.0.10/contrib), and take special care with `ProxyPreserveHost ` (apache) or `X-Forwarded-Host` (nginx).

Netbox puts absolute URLs in the image, rather than relative ones, so that you can copy the image to another server and the links will still work.  But in order to do that, it needs to "see" the correct hostname and port being used in the original request - and that's what those headers are for.
